game-core/src/main/java/games/strategy/engine/lobby/moderator/toolbox/tabs/BadWordsTabModel.java

```diff
@@ -27,15 +31,30 @@ String removeBadWord(final String wordToRemove) {
             .build());
   }
 
-  List<String> getBadWords() {
-    return moderatorToolboxClient.getBadWords(ClientSetting.moderatorApiKey.getValueOrThrow());
-  }
-
+  // TODO: test-me
   String addBadWord(final String newBadWord) {
     return moderatorToolboxClient.addBadWord(
         UpdateBadWordsArg.builder()
             .apiKey(ClientSetting.moderatorApiKey.getValueOrThrow())
             .badWord(newBadWord)
             .build());
   }
+
+  // TODO: test-me
+  /**
+   * Returns a "2 x N " list of lists. The first column value is the bad word value from the bad words table,
+   * the second column value is the "remove" button.
+   */
+  List<List<String>> getTableData() {
```

Comment:
I must admit that I'm not really convinced this is the best data structure for a table, but it works so whatever

(To clarify, the column amount is usually fixed, so a different inner type is usually the way to go, which also supports typisation etc.)